src/main/resources/icon_associations.xml

```diff
@@ -323,6 +323,7 @@
         <regex name="Swift" pattern=".*\.swift$" icon="/icons/files/swift.svg"/>
         <regex name="Swig" pattern=".*\.swig$" icon="/icons/files/swig.svg"/>
         <regex name="Symfony" pattern=".*\.symfony" icon="/icons/files/symfony.svg"/>
+        <regex name="Tailwind CSS Config" pattern="^tailwind(\..+)?\.js$" icon="/icons/files/tailwindcss.svg"/>
```

Comment:
is it always js? no json? .tailwindrc or something?

Reply:
or ts?

Reply:
No json. ts maybe sometime in the future. I adjusted the regex accordingly.